docs/source/api.rst

```diff
@@ -0,0 +1,60 @@
+*****************
+API Documentation
+*****************
+
+.. currentmodule:: merlin.systems
+
+
+Workflow Graph Constructors
```

Comment:
`Workflow` here should be `Ensemble`—which are very similar, so you have the right general idea, but this aligns the terminology with Triton.